ladybug/epw.py

```diff
@@ -263,19 +265,16 @@ def save(self, folder=None, file_name=None):
             modEpwFile.writelines(self._header)
             lines = []
             try:
+                # move first item to end position for fields on the hour
+                for field in range(0, self._num_of_fields):
+                    middle_hour = EPWFields.field_by_number(field).middle_hour
+                    if middle_hour is False:
+                        self._data[field].append(self._data[field].pop(0))
```

Comment:
Shouldn't we put the data back in order after this change? If I understand what's going on here correctly if one writes the file a couple of time each time the list will be shifted for one item.

Also you want to have the original epw object after writing the modified epw file.

Reply:
Good catch and that's a bug uncaught by our tests.  Let me know if you think that editing and fixing the epw object before and afterwards is the best approach (as opposed to trying to changing the data written into the file as it is being iterated through).  Seeing what you did below, I would guess that you would say editing it and then putting it back is conceptually clearer.